snippets/language-javascript.cson

```diff
@@ -104,3 +104,9 @@
   'Class':
     'prefix': 'class'
     'body': 'class ${1:ClassName} {\n\tconstructor($2) {\n\t\t$3\n\t}\n}'
+  'Exports Function':
```

Comment:
If we're adding an Exports snippet it might be worthwhile to add an `import` one as well.